examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
Hi Robert

Thanks for getting to this so quickly.

To start with I did try using osg_Color in the shader programs but osg Shape and Text use a single color vertex with the BIND_OVERALL mode. This isn't supported on GLES you have to have a color value per vertex. So I went for this color uniform approach which isn't ideal but at  it works.

I'm struggling to find a reference that states the BIND_OVERALL won't work, but i'm not 100% on what it does in terms of OpenGL calls, had a quick dig but got lost in array dispatcher land.